exercises/saddle-points/src/test/kotlin/MatrixTest.kt

```diff
@@ -0,0 +1,80 @@
+import org.junit.Ignore
+import org.junit.Test
+import java.util.Collections.emptySet
+import kotlin.test.assertEquals
+
+/*
+ * version: 1.1.0
+ */
+class MatrixTest {
+
+    @Test
+    fun testCanIdentifySingleSaddlePoint() {
+        val matrix = Matrix(listOf(
+            listOf(9, 8, 7),
+            listOf(5, 3, 2),
+            listOf(6, 6, 7)
+        ))
+
+        val expectedSaddlePoints = setOf(MatrixCoordinate(1, 0))
+
+        assertEquals(expectedSaddlePoints, matrix.saddlePoints)
+    }
+
+    //@Ignore("Remove to run test")
```

Comment:
Whoops, they should do. Those are just left in by mistake as I was re-running the tests to check the last change, I think!